llvm/include/llvm/CodeGen/ISDOpcodes.h

```diff
@@ -1294,6 +1294,11 @@ enum NodeType {
   MLOAD,
   MSTORE,
 
+  // Masked compress - consecutively place vector elements based on mask
+  // e.g., vec = {A, B, C, D} and mask = 1010
+  //         --> {A, C, ?, ?} where ? is undefined
+  MCOMPRESS,
```

Comment:
Please document the operand order

Reply:
Done. Is this enough? I've followed the examples of the other instructions.

Reply:
Expand the name to exactly match the intrinsic name